commands/checker.js

```diff
@@ -13,14 +13,9 @@ import { ethers, formatEther } from 'ethers'
 import { runUpdateRewardsLoop } from '../lib/rewards.js'
 import { runUpdateContractsLoop } from '../lib/contracts.js'
 
-const {
-  FIL_WALLET_ADDRESS,
-  PASSPHRASE
-} = process.env
+const { FIL_WALLET_ADDRESS, PASSPHRASE } = process.env
 
-const runtimeNames = [
-  'zinnia'
-]
+const runtimeNames = ['zinnia']
```

Comment:
Interesting, didn't we want things like this to be multi line arrays?

Reply:
Do we need to set the threshold to 0?

Reply:
I tried to set the threshold to `0`, but that made other lines look ugly. I will add a commit to show the result.

Reply:
Reverted in https://github.com/CheckerNetwork/node/pull/710/commits/1c1ff2b0e751545587c8cfbb14a11447d112f16d